xml/tuning_cgroups.xml

```diff
@@ -286,246 +286,225 @@ TasksMax=16284
    </para>
   </sect2>
  </sect1>
- <sect1>
-  <title>Controlling I/O with proportional weight policy</title>
-
-  <para>
-   This section introduces using the Linux kernel's block I/O controller to
-   prioritize I/O operations. The cgroup blkio subsystem controls and monitors
-   access to I/O on block devices. State objects that contain the subsystem
-   parameters for a cgroup are represented as pseudo-files within the cgroup
-   virtual file system, also called a pseudo-file system.
-  </para>
 
-  <para>
-   The examples in this section show how writing values to some of these
-   pseudo-files limits access or bandwidth, and reading values from some of
-   these pseudo-files provides information on I/O operations. Examples are
-   provided for both cgroup-v1 and cgroup-v2.
-  </para>
+<!-- BEGIN NEW VERSION -->
+<sect1 xml:id="sec-control-io-cgroups">
+<title>I/O control with cgroups</title>
+<para>
+This section introduces using the Linux kernel's block I/O controller to
+prioritize or throttle I/O operations,
+it leverages the means provided by systemd to configure cgroups.
+Finally, it discusses probable pitfalls when dealing with proportional I/O control.
+</para>
 
-  <para>
-   You need a test directory containing two files for testing performance and
-   changed settings. A quick way to create test files fully populated with text
-   is using the <command>yes</command> command. The following example commands
-   create a test directory, and then populate it with two 537 MB text files:
-  </para>
+<sect2 xml:id="sec-cgroups-prerequisites">
+<title>Prerequisites</title>
+<para>
+The following subsections describe steps that you must take in advance when
+you design and configure your system, since those aspects cannot be changed
+during runtime.
+</para>
 
-<screen>&prompt.plain-root;mkdir /io-cgroup
-&prompt.plain-root;cd /io-cgroup
-&prompt.plain-root;yes this is a test file | head -c 537MB > file1.txt
-&prompt.plain-root;yes this is a test file | head -c 537MB > file2.txt
-</screen>
-
-  <para>
-   To run the examples open three command shells. Two shells are for reader
-   processes, and one shell is for running the steps that control I/O. In the
-   examples, each command prompt is labeled to indicate if it represents one of
-   the reader processes, or I/O.
-  </para>
-
-  <sect2>
-   <title>Using cgroup-v1</title>
-   <para>
-    The following proportional weight policy files can be used to grant a
-    reader process a higher priority for I/O operations than other reader
-    processes accessing the same disk.
-   </para>
+<sect3 xml:id="sec-cgroups-filesystems">
+<title>Filesystem</title>
+<para>
+You must use a cgroup-aware filesystem. The recommended SLES
```

Comment:
Technically, it's "cgroup-writeback-aware" (sounds terrible).

Here's bacgrkound to what I meant. We can speak of two types of I/O: direct (can be both reader or writer) vs writeback. The I/O control of the former doesn't need special support of the filesystem, however, the latter requires the FS support.